### PR TITLE
Regenerate README for `db size`

### PR DESCRIPTION
**Pull request by danielbachhuber**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -350,6 +350,92 @@ Defaults to all tables registered to the $wpdb database handler.
...
+### wp db size
+
+Display the database name and size.
+
+~~~
+wp db size [--size_format] [--tables] [--format] [--scope=<scope>] [--network] [--all-tables-with-prefix] [--all-tables]
+~~~
+
+Display the database name and size for `DB_NAME` specified in wp-config.php,
+defaults to a human-readable number.
```
Comment (gitlost) [inline README.md]: Maybe "...wp-config.php. The size defaults..."

File: README.md [hunk 2]
```diff
@@ -350,6 +350,92 @@ Defaults to all tables registered to the $wpdb database handler.
...
+
+**OPTIONS**
+
+	[--size_format]
+		Display only the database size as a number.
```
Comment (gitlost) [inline README.md]: Maybe "Display the database size only, as a bare number." or similar.

File: README.md [hunk 3]
```diff
@@ -350,6 +350,92 @@ Defaults to all tables registered to the $wpdb database handler.
...
+    | wp_options            | 32 KB |
+    | wp_postmeta           | 48 KB |
+    | wp_terms              | 48 KB |
+    | wp_term_taxonomy      | 48 KB |
+    | wp_term_relationships | 32 KB |
+    | wp_termmeta           | 48 KB |
+    | wp_commentmeta        | 48 KB |
+    +-----------------------+-------+
+
+	   $ wp db size --size_format=b
```
Comment (gitlost) [inline README.md]: The tabbing is off here.

**Comment by danielbachhuber** [comment]
@gitlost Updated

**Pull request merged**